README.md

```diff
@@ -55,7 +55,7 @@ We suggest you use a secret to hide to app-code:
 - uses: chromaui/action@v1
   with:
     token: ${{ secrets.GITHUB_TOKEN }}
-    appCode: ${{ secrets.CHROMATIC_APP_CODE }}
+    projectToken: ${{ secrets.CHROMATIC_APP_CODE }}
```

Comment:
I think we should change the env var too?

Reply:
true